### PR TITLE
docs: add note about installing direnv with pixi

### PR DESCRIPTION
**Pull request by wolfv**
Hey @pavelzw I recently made sure that `direnv` is available for all conda-forge platforms :) Maybe you have additional thoughts on the commands to run for installation?

### CODE REVIEW

**Comment by pavelzw** [comment]
Did you try whether all this also works on Windows? I tried it out once, hit a roadblock (i think while installing direnv; something with powershell permissions iirc) and didn't investigate further.

**Comment by wolfv** [comment]
Didn't test Windows. I just made sure that the cross-compiled osx-arm64 build works fine.

File: docs/environment.md [hunk 1]
```diff
@@ -103,6 +103,15 @@ $ (default) which python
 
 ### Using `pixi` with `direnv`
 
+??? note "Installing direnv"
+
+    Of course you can use `pixi` to install `direnv` globally. We recommend to run
+
+    ```
+    pixi global install direnv```
```
Comment (pavelzw) [inline docs/environment.md:111]:
```suggestion
    ```shell
    pixi global install direnv
    ```
```

**Comment by pavelzw** [comment]
We might also want to link to this: https://direnv.net/docs/hook.html

**Comment by ruben-arts** [comment]
I can't get it to work on my windows machine. Their hook doens't seem to work for me. Even why I install it with winget. So not sure how to continue testing. Do you guys have the ability to test it on windows.

**Comment by wolfv** [comment]
I think this is a purely positive change - Windows support can be tackled (or not) at another point in time :)

**Pull request merged**